cylc/flow/network/schema.py

```diff
@@ -88,7 +88,7 @@ def parse_node_id(item, node_type=None):
     """Parse definition, job, or proxy id argument returning components.
 
     Args:
-        item: An string representing a node ID. Jobs fill out
+        item (str): An string representing a node ID. Jobs fill out
```

Comment:
Two things:
- Self correction, should be: `A string representing. . .`
- Why are you getting rid of ` (str)`? it's in the same style as `network/server.py` end-point methods?

Maybe I don't understand docstrings properly!

Reply:
>Self correction, should be: A string representing. . .

Ops! Correcting now.

>Why are you getting rid of (str)? it's in the same style as network/server.py end-point methods?

Hmm, I thought I was adding it, not getting rid

Reply:
> > Self correction, should be: A string representing. . .
> 
> Ops! Correcting now.
> 
> > Why are you getting rid of (str)? it's in the same style as network/server.py end-point methods?
> 
> Hmm, I thought I was adding it, not getting rid

Double Oppsie! Yes please add!